src/javaRosa/util.js

```diff
@@ -252,7 +252,8 @@ define([
     var insertOutputRef = function(vellum, target, path, mug, dateFormat) {
         var output = getOutputRef(path, dateFormat),
             form = vellum.data.core.form;
-        if ((!mug && vellum.opts().features.rich_text) || (mug && mug.supportsRichText())) {
+        if ((!mug && form.useRichText !== false) ||
```

Comment:
Sorry, original comment wasn't really relevant - forgot that https://github.com/dimagi/Vellum/pull/667/files#diff-62acef2590804164cb77b4b5aced00ebR141 initializes it but hasn't been merged yet.